src/c++/perf_analyzer/constants.h

```diff
@@ -31,6 +31,8 @@
 
 namespace triton { namespace perfanalyzer {
 
+const std::string VERSION{"2.36.0dev / 23.07dev"};
```

Comment:
Is this the standard way to do this? Aren't these already defined somewhere like as cmake args?

Reply:
Standard way is to create a <PROJECT_NAME>_VERSION file with the value. This is what server does but they also have python components that lead to this approach. Server does not output a version number via CLI. Instead, it is printed at the start of the container. 
![Screenshot 2023-06-21 at 09 36 45](https://github.com/triton-inference-server/client/assets/6505145/1ae68014-a715-4dad-b973-16f152b80e72)

Reply:
In DLProf I'm pretty sure we used cmake defines (that were fed in through gitlab variables). Any reason we can't stick to that?

I don't think Misha currently changes anything in the clients repo for the release.  Having a hardcoded string in the code would require him to make two extra commits every time we freeze.

Reply:
Added plumbing to get the version number from a cmake define. 
To use it, we need to add the version value to the cmake command here: https://github.com/triton-inference-server/server/blob/e75371b24b75f6dd1dde4c9b78a4e8614ac8c0db/Dockerfile.sdk#L136

Reply:
Very cool!

@mc-nv can you look at the PR and confirm things look ok?

Reply:
Suppose Perf Analyzer version will keep releasing aligned to Triton Server version. 
There why I would ask to assign by default version of the Perf Analyzer equal to Triton Server

Reply:
I will pin those two together by default but leave the option to set PA version if we need.